project-base/storefront/components/Blocks/OpeningHours/OpeningStatus.tsx

```diff
@@ -1,24 +1,43 @@
 import { Flag } from 'components/Basic/Flag/Flag';
+import { TypeStoreOpeningStatusEnum } from 'graphql/types';
 import useTranslation from 'next-translate/useTranslation';
 import { twMergeCustom } from 'utils/twMerge';
 
-export const OpeningStatus: FC<{ isOpen: boolean; isDynamic?: boolean }> = ({
-    isOpen,
+export const OpeningStatus: FC<{ status: TypeStoreOpeningStatusEnum; isDynamic?: boolean }> = ({
```

Comment:
To maintain consistency across the codebase, I recommend outsourcing props like this: 
`type OpeningStatusProps = { status: TypeStoreOpeningStatusEnum; isDynamic?: boolean; }`.

Reply:
Fixed